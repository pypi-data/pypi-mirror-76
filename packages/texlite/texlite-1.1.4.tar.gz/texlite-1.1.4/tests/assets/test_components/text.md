Here is some text. Nunc convallis auctor risus, vitae commodo dolor hendrerit nec. Suspendisse tempus pellentesque mattis. In erat velit, fringilla aliquet ullamcorper eget, elementum non dolor. Aliquam erat volutpat.

Maecenas eget pellentesque sem. Duis aliquet augue congue, cursus elit eu, sagittis lacus. 

Lorem ipsum dolor sit amet, consectetur adipiscing elit.

---

Here is a symbol that needs to be converted to something safe: &.


This is **bold**.
This is *italics*.
This is `in-line code`.
This is [link](https://github.com).

Here is **bold text** and **here is some more**.
Here is *italics text* and *here is some more*.
Here is `in-line code` and `here is some more`.
Here is "a quotation" and "here is some more".
Here is [a hyperlink](http://github.com) and [here is another](https://google.com).

This is **not bold text
This is **not bold text **
This is ** not bold text**
This is not bold text**

This is *not italics
This is *not italics *
This is * not italics*
This is not italics*

This is `not in-line code
This is not in-line code`

This is "not a quotation
This is not a quotation"

This is [not a link]
This is [not a link)