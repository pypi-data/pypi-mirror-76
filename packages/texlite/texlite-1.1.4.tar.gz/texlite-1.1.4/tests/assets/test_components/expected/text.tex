% meta
\documentclass[10pt]{extarticle}

% packages
\usepackage[utf8]{inputenc}
\usepackage[margin=1.6in]{geometry}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{amssymb}

% preamble commands
\linespread{1.0}
\usepackage{graphicx}
\graphicspath{{tests/assets/test_components/}}

% document details
\date{}

% ------------------------------------------------------------------------------
\begin{document}
% ------------------------------------------------------------------------------

Here is some text. Nunc convallis auctor risus, vitae commodo dolor hendrerit nec. Suspendisse tempus pellentesque mattis. In erat velit, fringilla aliquet ullamcorper eget, elementum non dolor. Aliquam erat volutpat.

Maecenas eget pellentesque sem. Duis aliquet augue congue, cursus elit eu, sagittis lacus. 

Lorem ipsum dolor sit amet, consectetur adipiscing elit.

\medskip

Here is a symbol that needs to be converted to something safe: \&.

This is \textbf{bold}.

This is \textit{italics}.

This is \texttt{in-line code}.

This is \href{https://github.com}{link}.

Here is \textbf{bold text} and \textbf{here is some more}.

Here is \textit{italics text} and \textit{here is some more}.

Here is \texttt{in-line code} and \texttt{here is some more}.

Here is ``a quotation" and ``here is some more".

Here is \href{http://github.com}{a hyperlink} and \href{https://google.com}{here is another}.

This is **not bold text

This is **not bold text **

This is ** not bold text**

This is not bold text**

This is *not italics

This is *not italics *

This is * not italics*

This is not italics*

This is `not in-line code

This is not in-line code`

This is "not a quotation

This is not a quotation"

This is [not a link]

This is [not a link)

% ------------------------------------------------------------------------------
\end{document}
% ------------------------------------------------------------------------------

